pom.xml

```diff
@@ -1,153 +1,212 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.redhat.syseng.soleng.rhpam</groupId>
-	<artifactId>process-migration-service</artifactId>
-	<name>Process Instance Migration Service</name>
-	<version>1.0.0-SNAPSHOT</version>
-	<packaging>war</packaging>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.redhat.syseng.soleng.rhpam</groupId>
+    <artifactId>process-migration-service</artifactId>
+    <name>Process Instance Migration Service</name>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>war</packaging>
 
-	<properties>
-		<thorntail.version>2.0.0.Final</thorntail.version>
-		<kie.version>7.8.0.Final</kie.version>
-		<fabric8-maven.version>3.5.41</fabric8-maven.version>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
-		<failOnMissingWebXml>false</failOnMissingWebXml>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-	</properties>
+    <properties>
+        <thorntail.version>2.0.0.Final</thorntail.version>
+        <kie.version>7.8.0.Final</kie.version>
+        <fabric8-maven.version>3.5.41</fabric8-maven.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <version.h2>1.4.187</version.h2>
+        <version.mysql>8.0.12</version.mysql>
+        <version.postgresql>42.2.5</version.postgresql>
```

Comment:
Remove

Reply:
why do you want to remove the version number of JDBC driver?

Reply:
Because we are only providing mysql or postgresql not both as a default implementation, aren't we?